site/src/components/layout/layout.md

```diff
@@ -81,3 +81,17 @@ svg.append('g')
       'flexDirection': 'row'
   });
 ```
+
+When building high performance charts e.g. when targeting 60fps, layout can be a source of performance problems. So in order to get the best performance, it's worth understanding some of its internals.
+
+At a high level, when a node is laid-out -
```

Comment:
You probably need to make it clear that you are referring to node layout within the context of our SVG / flexbox implementation.